dev-docs/source/UserSupport.rst

```diff
@@ -205,8 +205,18 @@ For a full release, ``/opt/Mantid/`` is likely the correct install path. Take ca
 
 11. Further diagnosis for process monitoring: `strace <https://strace.io/>`_.
 
+12. To get a stacktrace from C++ follow these steps:
+
+- Before opening MantidWorkbench, open a terminal and run ``ulimit -c unlimited``
+- Then in the same terminal, open MantidWorkbench with ``/opt/mantidworkbench/bin/mantidworkbench`` and use as normal.
+- If there is a crash, then you will find a ``core.1234`` (some number) file
+  in the current working directory of the terminal, which the user can send to us!
+- Run ``gdb python3 core.1234`` to get the stacktrace from this coredump file.
```

Comment:
```suggestion
    - Before opening MantidWorkbench, open a terminal and run ``ulimit -c unlimited``
    - Then in the same terminal, open MantidWorkbench with ``/opt/mantidworkbench/bin/mantidworkbench`` and use as normal.
    - If there is a crash, then you will find a ``core.1234`` (some number) file
      in the current working directory of the terminal, which the user can send to us!
    - Run ``gdb python3 core.1234`` to get the stacktrace from this coredump file.
```

Reply:
Thanks, this is exactly what I wanted! the next step 13 was a little close to the end of the list so I've added some pipe characters to help with spacing!

Reply:
The pipe characters seem to un-indent bullet point 13 but I do think it looks better regardless with that extra space. I've had a go at finding a way to indent properly with the same spacing but have had no luck.

Reply:
I've also played around with all the options and I've now just swapped the order of 12 and 13 so it looks right! I have also added pipe characters before each OS section so tell me if you think that renders well or not!